### PR TITLE
Update Swagger with Site Auth changes

### PR DESCRIPTION
**Pull request by mkarmark**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json [hunk 1]
```diff
@@ -7018,37 +7117,36 @@
           "200": {
             "description": "OK",
             "schema": {
-              "$ref": "#/definitions/PublicCertificateCollection"
+              "type": "file"
```
Comment (ravbhatnagar) [inline specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json:7120]: only application/json is supported.

File: specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json [hunk 2]
```diff
@@ -6982,28 +7077,32 @@
...
             }
           }
-        },
-        "x-ms-pageable": {
-          "nextLinkName": "nextLink"
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/publicCertificates": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/processes/{processId}/dump": {
```
Comment (ravbhatnagar) [inline specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json:7083]: does this need to be on control plane? what can be the size of this dump? better way would be to put this dump on azure storage blob and return the pointer of that blob to the customer.

File: specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json [hunk 3]
```diff
@@ -8757,22 +8855,129 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/config/azurestorageaccounts": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/config/authsettingsV2": {
```
Comment (ravbhatnagar) [inline specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json:8858]: same comments as for site/config

File: specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json [hunk 4]
```diff
@@ -8757,22 +8855,129 @@
...
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/config/authsettingsV2/list": {
```
Comment (ravbhatnagar) [inline specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json:8916]: why do you need a /list for a GET API

**Comment by ravbhatnagar** [comment]
No response to the ARM review yet @mkarmark is this PR still valid?

**Comment by mkarmark** [comment]
Sorry for the confusion, didn't realize that I didn't close this PR.

**Pull request closed without merging**